metadata-models/src/main/pegasus/com/linkedin/dataset/FieldLineageDetails.pdl

```diff
@@ -0,0 +1,18 @@
+namespace com.linkedin.dataset
+
+import com.linkedin.common.Urn
+
+/**
+ * Details of the parent fields and transformation resulting in new field.
+ */
+record FieldLineageDetails {
+    transform: optional string
+
+    parentFields: array[Urn]
+
+/*
+  Note that information of a particular lineage can be provided manually by a person on the UI, or sourced from another data platform, or determined by Datahub itself, or a combination of these.
+  The information provider details can be modeled here e.g. user-id, dataPlatformUrn etc.
+  TODO
```

Comment:
Missing TODO?

Reply:
To be done in next iteration/release when required.